root/static/scripts/common/constants/locales.json

```diff
@@ -97,8 +98,6 @@
   "ckb_IR":"Central Kurdish Iran",
   "cs":"Czech",
   "cs_CZ":"Czech Czechia",
-  "cu":"Church Slavic",
-  "cu_RU":"Church Slavic Russia",
```

Comment:
Any idea why these were dropped? We certainly have works written in Church Slavic although surprisingly we don't have work alias using these at the moment.

Reply:
Yes, see the below comment about “Volapük”.